index.js

```diff
@@ -37,6 +52,17 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 4", "color: red") 
 
+function sameSameButDifferent(num, maybeNum) {
+  if (num === maybeNum) {
+    return 'same same'
+  } else if { num == maybeNum) {
+    return 'same same (but different)'
+  } else {
+    return "different"
+  }
+    
+}
+
```

Comment:
Nice! Another solution would be to convert the string to a number with parseInt to avoid `==` :

`} else if (num === parseInt(maybeNum)) {`